### PR TITLE
Bme sensor integration

### PR DESCRIPTION
**Pull request by Tooseriuz**

### CODE REVIEW

File: lib-core/src/main/java/org/rfcx/guardian/utility/rfcx/RfcxPrefs.java [hunk 1]
```diff
@@ -149,6 +149,8 @@ public class RfcxPrefs {
                 put(Pref.ADMIN_BLUETOOTH_FUNCTION, "off");
                 put(Pref.ENABLE_FILE_SOCKET, "false");
 
+                put(Pref.ENABLE_SENSOR_BME688, "true");
+                put(Pref.PING_SENSOR_VALUES, "bme");
```
Comment (antonyharfield) [inline lib-core/src/main/java/org/rfcx/guardian/utility/rfcx/RfcxPrefs.java:153]: Is this the default? The default needs to be "off".

File: role-admin/src/main/java/org/rfcx/guardian/admin/device/i2c/DeviceI2CUtils.java [hunk 2]
```diff
@@ -24,52 +24,47 @@ public static void setSentinelLoggingVerbosity(Context context) {
...
         JSONArray sensorJsonArray = new JSONArray();
         try {
             JSONObject sensorJson = new JSONObject();
-            sensorJson.put("environment", app.sentinelSensorDb.dbEnvironment.getConcatRowsWithLabelPrepended("environment"));
-            sensorJson.put("battery", app.sentinelSensorDb.dbBattery.getConcatRowsWithLabelPrepended("battery"));
-            sensorJsonArray.put(sensorJson);
 
+            String bmeValues = app.sentrySensorDb.dbBME688.getConcatRowsWithLabelPrepended("bme688");
+            if (!bmeValues.split("\\*")[2].equalsIgnoreCase("0")) {
+                sensorJson.put("sentinel_sensor", bmeValues);
```
Comment (antonyharfield) [inline role-admin/src/main/java/org/rfcx/guardian/admin/device/i2c/DeviceI2CUtils.java:34]: Do we need to change this to be "s"?

**Review by antonyharfield — APPROVED** [review]

File: lib-core/src/main/java/org/rfcx/guardian/utility/rfcx/RfcxPrefs.java (hunk 1 above)
Comment (Tooseriuz) [reply lib-core/src/main/java/org/rfcx/guardian/utility/rfcx/RfcxPrefs.java:153]: I have set it to be `false` in the [infineon PR](https://github.com/rfcx/guardian-software/pull/222/files#diff-6ce0716ee024762a706e4092d79b3a8e60e769c2c18718cd5742e6dd30232936R152-R154) and remove the `ping_sensor_values` pref

File: role-admin/src/main/java/org/rfcx/guardian/admin/device/i2c/DeviceI2CUtils.java (hunk 2 above)
Comment (Tooseriuz) [reply role-admin/src/main/java/org/rfcx/guardian/admin/device/i2c/DeviceI2CUtils.java:34]: it is in [infineon PR](https://github.com/rfcx/guardian-software/pull/222/files#diff-4866df8d44b0950f474dfc80d5592bd59dbdb6e71911d2a2ed6cf9bb283e579eR190-R197)

**Pull request merged**